benchmarks/ner_embeddings/README.md

```diff
@@ -0,0 +1,79 @@
+<!-- SPACY PROJECT: AUTO-GENERATED DOCS START (do not remove) -->
+
+# 🪐 spaCy Project: Comparing embedding layers in spaCy
+
+This project compares `MultiHashEmbed` with its standard embedding counterpart
+`MultiEmbed` for Named Entity Recognition. In order to download and preprocess the datasets,
+you first need to run the commands in the [span-labeling-datasets](https://github.com/explosion/projects/tree/v3/benchmarks/span-labeling-datasets) project. For example, let's perform the
+conversion command for the [Anatomical Entity Mention (AnEM)](http://www.nactem.ac.uk/anatomy/) corpus:
```

Comment:
As noted somewhere else, it would be good to not have to rely on a different project and have example files readily available for anyone to test